src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const lines = sourceString.split(';');
+  const cssProperties = {};
+
+  lines.forEach((line) => {
+    const trimmedLine = line.trim();
+
+    if (trimmedLine === '' || !trimmedLine.includes(':')) {
+      return;
+    }
 
+    const [property, value] = trimmedLine.split(':').map((part) => part.trim());
+
+    if (property && value) {
+      cssProperties[property.trim()] = value.trim();
+    }
```

Comment:
Using ``trim()`` in `map` already removes extra spaces, so an additional call to trim() for property and value is unnecessary.

Example:

```suggestion
    if (property && value) {
      cssProperties[property] = value;
    }
```